docs/post-upgrade/template-script.asciidoc

```diff
@@ -2,13 +2,15 @@
 == Index template script
 
 This code creates a new index template for temporarily storing existing
-detection alerts when you update `.siem-signals-*` index mappings. You
+detection alerts when you update the index mappings for detection alert indices. You
 need to update the index mappings to visualize process relationships after
 upgrading to {stack} release 7.9.0 or 7.9.1 from a previous minor release
 (7.8.x, 7.7.x, and so on).
 
+IMPORTANT: As of {stack} version 8.0.0 and newer, the system index for detection alerts has been renamed from `.siem-signals-*` to `.alerts-security.alerts-*`. 
```

Comment:
Instead of `.siem-signals-*` and `.alerts-security.alerts-*` I'd recommend using `.siem-signals-<Kibana space>` and `.alerts-security.alerts-<Kibana space>` similar to how it's used above.

Reply:
```suggestion
IMPORTANT: As of {stack} version 8.0.0 and newer, the system index for detection alerts has been renamed from `.siem-signals-<Kibana space>` to `.alerts-security.alerts-<Kibana space>`. 
```